### ISSUE TITLE
Загрузка выписки с отрицательными суммами

### ISSUE THREAD

**Issue by gretis** [shcherbakov-vadim/1c-money-custom-issues#30]
Если в выписке встречается отрицательный расход, он определяется корректно, но сама операция распознается не как списание (сторнирующая операция), а как перемещение

**Comment by shcherbakov-vadim** [comment]
Нужен пример того, как отрицательный расход становится перемещением

**Comment by gretis** [comment]
![image](https://github.com/user-attachments/assets/589f3c81-3a06-441d-9f60-4c5dd7a755ea)
![image](https://github.com/user-attachments/assets/ff73f138-80df-41ee-99d9-feb382a9a148)
[zendump(2024-12-11)_2023-07-07-2023-07-07_out.csv](https://github.com/user-attachments/files/18123806/zendump.2024-12-11._2023-07-07-2023-07-07_out.csv)

**Comment by shcherbakov-vadim** [comment]
Для второй строки в качестве получателя указан кошелек "Наличные Натальи", поэтому операция воспринимается как перемещение. Это типовая функциональность

**Comment by gretis** [comment]
> указан кошелек

НЕТ, именно в этом и есть ошибка. В загружаемой строка таких данных НЕТ. Это видно на скриншоте, кстати.

**Comment by shcherbakov-vadim** [comment]
То есть вы не понимаете каким образом у вас во второй строке установился кошелек "Наличные Натальи"? Нужно правила обработки смотреть и содержимое регистра сведений "Ключевые слова объектов". Возможно в нем для "Приход / ⚖️ возврат, возмещение" установлен этот кошелек. В любом случае это проблема идентификации аналитик, а не подбора некорректного вида операции

**Comment by gretis** [comment]
Я удалил все упоминания, которые могли бы повлиять в РС "Ключевые слова объектов". Но это не помогло. Какие данные необходимо собрать для уточнения?

**Comment by gretis** [comment]
А вот и пример созрел. Вот так распозналось "по умолчанию"

![Image](https://github.com/user-attachments/assets/86e0e9fc-62f3-4f2d-b917-b76fdace2590)

При этом вот, по сути, подтверждение того, что в РС нет такого сопоставления, но есть другое, которое почему-то игнорируется

![Image](https://github.com/user-attachments/assets/abfd0608-93a0-45bd-aa31-726e8962363b)

**Comment by shcherbakov-vadim** [comment]
Судя по скриншоту и окну диалога вы вместо статьи расходов в строке указали долг. Но почему вы то что изображено на скриншоте называете "распозналось по-умолчанию"?

**Comment by gretis** [comment]
> почему

потому что я ничего еще не указывал. Я только выбрал файл для загрузки. При том, что 
>Я удалил все упоминания, которые могли бы повлиять в РС "Ключевые слова объектов"

**Comment by shcherbakov-vadim** [comment]
Просьба воспроизвести проблему на демобазе и по шагам описать порядок ее воспроизведения
